### ISSUE TITLE
Need use partial helper

### ISSUE THREAD

**Issue by jiangtj** [jiangtj/hexo-theme-plus#1]
https://hexo.io/zh-cn/docs/helpers#partial

**Comment by jingxinxin** [comment]
Can you add not only replace files, but also new files?
I need custom new *.swig files

**Comment by jiangtj** [comment]
@jingxinxin 
Yes, you can add new *.swig files as layout, but you must load it by partial.
https://hexo.io/zh-cn/docs/helpers#partial

This plugin only use `hexo.theme.setView`, see https://hexo.io/zh-cn/api/themes

**Comment by jingxinxin** [comment]
@JiangTJ But I don't wanna modify the files in theme path. Can I add new *.swig files to the path of source/_data?

**Comment by jiangtj** [comment]
I saw your PR in NexT. So I may understand your purpose.
Although I wrote this plugin for NexT at first, it doesn't work well now.
Because NexT uses `include` `import` and `extends`, not `partial`.

You can try copy `page.swig` to outside.  Modify this `page.swig` to include/partial the new layout.
You will encounter some path error and have to fix it.
